tasks/setup-direct.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+- apt: name=apt-transport-https state=installed update_cache=yes
+
+# wget -O - http://download.gluster.org/pub/gluster/glusterfs/{{ gluster_version }}/rsa.pub | apt-key add -
+- name: Add the GPG key to apt
+  apt_key:
+    url: "http://download.gluster.org/pub/gluster/glusterfs/{{ gluster_version }}/rsa.pub"
+    state: present
+
+# echo deb http://download.gluster.org/pub/gluster/glusterfs/{{ gluster_version }}/LATEST/Debian/stretch/apt stretch main >
+#      /etc/apt/sources.list.d/gluster.list
+# Add specified repository into sources list using specified filename.
+- apt_repository:
+    repo: deb http://download.gluster.org/pub/gluster/glusterfs/{{ gluster_version }}/LATEST/Debian/stretch/apt stretch main
+    state: present
+    filename: 'gluster'
+
+# apt-get install [ glusterfs-server | glusterfs-client ]
+#- name: Update repositories cache and install "glusterfs client server" packages
+- apt: name="{{ item }}" state=installed update_cache=yes
```

Comment:
I found that name="{{ item }}={{ gluster_version }}*" helps a lot to ensure that the correct version gets installed, versus some older version that may be in the package cache or another repo from elsewhere. (At least I had to do that on my end to get it to work, it kept installing an ancient version from before instead)